app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
toán tử `||=` có tác dụng gì?

Reply:
Dạ toán tử `||=` được sử dụng để gán giá trị cho một biến chỉ khi biến đó chưa được gán giá trị hoặc là nil hoặc false. Nó kiểm tra xem biến có giá trị hay không, nếu biến không có giá trị hoặc là nil hoặc false, thì giá trị mới sẽ được gán vào biến, nếu không biểu thức sẽ không làm gì.